src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -722,9 +725,9 @@ namespace Opm {
             const std::string& wellNamePattern = record.getItem("WELL").getTrimmedString(0);
 
             for (auto* well : getWells(wellNamePattern)) {
-                // TODO: Can this be done like this? Setting the temperature only has an
-                // effect on injectors, but specifying this for producers won't hurt and
-                // wells can also switch their injector/producer status. Note that
+                // TODO: Is this the right approach? Setting the well temperature only
```

Comment:
Looks like this comment block is currently duplicated?

Reply:
it is duplicated, but that was on purpose because the statement applies equally for WTEMP and WINJTEMP as far as I can see.